pom.xml

```diff
@@ -74,6 +74,11 @@
             <version>4.11</version>
             <classifier>javadoc</classifier>
         </dependency>
+        <dependency>
+            <groupId>io.reactivex.rxjava2</groupId>
+            <artifactId>rxjava</artifactId>
+            <version>2.1.10</version>
+        </dependency>
```

Comment:
why is this here? just to suffer?

Reply:
hello from danielle

Reply:
Added this for Java CL running the tests